### PR TITLE
update new paths

### PR DESCRIPTION
**Pull request by mkangia**
https://dimagi-dev.atlassian.net/browse/ICDS-1178

Just updating to add new paths in the form.
A follow up would be to iterate forms submitted post release of the change which was on 26 November 2019

### CODE REVIEW

File: custom/icds_reports/ucr/tests/test_awc_mng_form_ucr.py [hunk 1]
```diff
@@ -16,38 +16,48 @@ def test_awc_visit_form_with_location_entered(self):
             [{'count': 1,
               'awc_location_long': None,
               'awc_not_open_aww_not_available': 0,
+              'awc_not_open_aww_not_available_v2': 1,
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/tests/test_awc_mng_form_ucr.py]: 👍  happy to see these test cases.

**Review by Rohit25negi — APPROVED** [review]

**Comment by calellowitz** [comment]
Just wanted to flag my comment from the ticket itself that I think the best way forward here might not be new columns, but changing the old columns to alternately accept the new paths. That will prevent having to rebuild the data source or change any of the reports. Apologies if that has already been rejected for other reasons, and let me know if you have any questions about that.

**Comment by mkangia** [comment]
@calellowitz @Rohit25negi 
Updated to modify the existing columns itself to avoid changing any reports that use this data source.
Thanks Cal for the suggestion to do so.

File: custom/icds_reports/ucr/tests/data/awc_visit_form_with_location.xml [hunk 2]
```diff
@@ -27,6 +27,11 @@
 		<hot_meal_served/>
 		<infrastructure_update/>
 	</center_questions>
+    <awc_details>
+        <center_questions>
+		    <awc_closed_reason>aww_helper_unavailable</awc_closed_reason>
+	    </center_questions>
+    </awc_details>
```
Comment (calellowitz) [inline custom/icds_reports/ucr/tests/data/awc_visit_form_with_location.xml]: Nice adding tests for both formats

**Review by calellowitz — APPROVED** [review]

**Pull request merged**